morpheus/messages/message_meta.py

```diff
@@ -80,6 +82,13 @@ def __init__(self, df: pd.DataFrame) -> None:
         self._mutex = threading.RLock()
         self._df = df
 
+    @classmethod
+    def make_from_file(cls, filename: str):
+        if (cls._should_use_cpp()):
+            return cls._cpp_class.make_from_file(filename)
+        else:
+            return MessageMeta(read_file_to_df(filename, file_type=FileTypes.Auto, df_type="cudf"))
```

Comment:
This determines too much for the user. They cant change the options to `read_file_to_df`. Whats the purpose of this function again? Is it used anywhere?

Reply:
The `MessageMetaInterfaceProxy` class has had this method:
```cpp
static std::shared_ptr<MessageMeta> init_cpp(const std::string& filename);
```

For some time, but it was never mapped to anything in the python bindings, so I figure I might as well go ahead and add the binding.

It has the advantage of being able to, from Python, deserialize a dataframe from file and construct a `MessageMeta` entirely in C++.

Without this, the user would have to do the file read in Python (`read_file_to_df`).